Decred Wallet/Features/Navigation Menu/MenuItem.swift

```diff
@@ -10,72 +10,44 @@ import UIKit
     case overview = "Overview"
-    case history = "History"
-    case send = "Send"
-    case receive = "Receive"
-    case accounts = "Accounts"
-    case security = "Security"
-    case settings = "Settings"
-    case help = "Help"
+    case transactions = "Transactions"
+    case accounts = "Wallets"
+    case more = "More"
     
-    // Each menu item's VC is wrapped in a navigation controller to enable the display of a navigation bar on each page,
+    // Each menu item's VC is wrapped in a navigation controller to enable the display of a navigation bar on each page
     // and to allow each page perform VC navigations using `self.navigationController?.pushViewController`.
     var viewController: UIViewController {
         switch self {
         case .overview:
             return Storyboards.Overview.instantiateViewController(for: OverviewViewController.self).wrapInNavigationcontroller()
             
-        case .history:
+        case .transactions:
             return TransactionHistoryViewController(nibName: "TransactionHistoryViewController", bundle: nil).wrapInNavigationcontroller()
-            
-        case .send:
-            return SendViewController.instance
-            
-        case .receive:
-            return Storyboards.Main.instantiateViewController(for: ReceiveViewController.self).wrapInNavigationcontroller()
-            
+        
         case .accounts:
             return Storyboards.Main.instantiateViewController(for: AccountViewController.self).wrapInNavigationcontroller()
-            
-        case .security:
-            return Storyboards.SecurityMenu.instantiateViewController(for: SecurityMenuViewController.self).wrapInNavigationcontroller()
-            
-        case .settings:
-            return SettingsController.instantiate().wrapInNavigationcontroller()
-            
-        case .help:
-            return Storyboards.Main.instantiateViewController(for: HelpViewController.self).wrapInNavigationcontroller()
+        
+        case .more:
+            return Storyboards.Overview.instantiateViewController(for: OverviewViewController.self).wrapInNavigationcontroller() // Change this when settings view controller is ready
         }
     }
     
     var icon: UIImage? {
         switch self {
-        case .overview:
-            return UIImage(named: "overview")
+            case .overview:
+                return UIImage(named: "nav_menu/ic_overview")
             
-        case .history:
-            return UIImage(named: "history")
+            case .transactions:
+                return UIImage(named: "nav_menu/ic_transactions")
             
-        case .send:
-            return  UIImage(named: "send")
+            case .accounts:
+                return UIImage(named: "nav_menu/ic_accounts")
             
-        case .receive:
-            return UIImage(named: "receive")
-            
-        case .accounts:
-            return UIImage(named: "menu-account")
```

Comment:
Seems like `menu-account` can be deleted.

Reply:
resolved in 37e2453